_projects/2024/100508856/100508856.Rmd

```diff
@@ -0,0 +1,333 @@
+---
+title: "The evolution of working hours in the world"
+description: |
+  This project consists in the replicaion and alternative representation of graphs from Our World in Data called "Working Hours"
+categories: "2023"
+author: Miao Liao
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: TRUE
+preview: original.png
```

Comment:
This is to show a image as preview in the gallery, but you didn't add a `original.png` file to the pull request. Please, remove this.